src/CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ set(GEN_SRCS "")
 add_custom_command(OUTPUT time_fmts.cc COMMAND ptimec ${TIME_FORMATS} >
         time_fmts.cc)
 
-add_library(lnavdt STATIC config.h.in ptimec.hh ptimec_rt.cc time_fmts.cc)
```

Comment:
there is typically no need to add header files to cmake libraries. `config.h.in` isn't actually present on a file system so cmake fails without this edit